vars/kola.groovy

```diff
@@ -1,18 +1,20 @@
 // Run kola tests on the latest build in the cosa dir
 // Available parameters:
-//    addExtTests:       []string -- list of test paths to run
-//    allowUpgradeFail   boolean  -- warn instead of fail on upgrade failure
-//    arch:              string   -- the target architecture
-//    cosaDir:           string   -- cosa working directory
-//    parallel:          integer  -- number of tests to run in parallel (default: # CPUs)
-//    skipBasicScenarios boolean  -- skip basic qemu scenarios
-//    skipSecureBoot     boolean  -- skip secureboot tests
-//    skipUpgrade:       boolean  -- skip running `cosa kola --upgrades`
-//    build:             string   -- cosa build ID to target
-//    platformArgs:      string   -- platform-specific kola args (e.g. '-p aws --aws-ami ...`)
-//    extraArgs:         string   -- additional kola args for `kola run` (e.g. `ext.*`)
-//    disableRerun:      boolean  -- disable reruns of failed tests
-//    marker:            string   -- some identifying text to add to uploaded artifact filenames
+//    addExtTests:          []string -- list of test paths to run
+//    allowUpgradeFail      boolean  -- warn instead of fail on upgrade failure
+//    allowRerunSuccess     boolean  -- whether or not to allowRerunSuccess
+//    allowRerunSuccessArgs string   -- args to pass to --allow-rerun-success
```

Comment:
We should mention the default for `allowRerunSuccessArg`. For `allowRerunSuccess`, should we make it e.g. `disallowRerunSuccess` for consistency with other "negative" options we have and so it's obvious what the default it?

Reply:
went with `disableRerunSuccess` here since there was another `disable` option (`disableRerun`).